CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changes
 
+## Master
```

Comment:
Pleaes remove this line

Reply:
Can we reword this to "next release"? I think it's valuable to mark the topmost block of changes in the changelog as "unreleased yet, but will be soon".

Reply:
Yes, I would replace it with `unreleased` or `next release`

Reply:
I updated it with Next Release